dev/README.md

```diff
@@ -326,6 +326,26 @@ act schedule -W .github/workflows/e2e-multicluster-ci.yml
 
 ### Troubleshooting
 
+#### fatal: not a git repository
+
+```shell
+get repo root in /tmp/foo: exit status 128: fatal: not a git repository
```

Comment:
This is the result of the improvement of the testing code in this PR. Without it, it just says `not a git repository`.